.thoth.yaml

```diff
@@ -0,0 +1,15 @@
+managers:
```

Comment:
You prepend also configuration for recommendations:

```yaml
host: test.thoth-station.ninja
tls_verify: false
requirements_format: pipenv

runtime_environments:
  - name: ubi8
    operating_system:
      name: ubi
      version: "8"
    python_version: "3.6"
    recommendation_type: latest
```

Reply:
EDIT: updated params

Reply:
Done :+1:

Reply:
:+1: